bot/exts/fun/bonker.py

```diff
@@ -0,0 +1,109 @@
+import asyncio
+import concurrent.futures
```

Comment:
```suggestion
from concurrent import futures
```
If we just need the `futures`, why not just do this?

Reply:
done